main.yml

```diff
@@ -352,8 +350,17 @@ settings:
   title: Cloud Services Settings
   frontend:
     sub_apps:
+      - id: ''
+        default: true
+        title: 'User Preferences'
```

Comment:
I think the mocks had this as "General"

https://marvelapp.com/5a4j976/screen/63049476